src/styles/blocks/card.scss

```diff
@@ -0,0 +1,121 @@
+.card {
+  width: 198px;
+  height: 406px;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+
+  &__product {
+    display: block;
+
+    &-img {
+      padding: 33px 20px 40px;
+      width: 160px;
+      height: 134px;
+      flex-shrink: 0;
+    }
+  }
+
+  &__name {
+    margin: 0;
+    padding: 0 16px;
+    color:#060B35;
```

Comment:
move all color that you reuse to variable

Reply:
done. but when I duplicated my card, the test showed a problem with the button. but it works.